c++/src/kj/async.c++

```diff
@@ -1154,14 +1170,14 @@ void waitImpl(Own<_::PromiseNode>&& node, _::ExceptionOrValue& result, WaitScope
 
     // The main stack switched back to us, meaning either the event we registered with
     // node->onReady() fired, or we are being canceled by FiberBase's destructor.
-
+    
```

Comment:
Nitpick: Added trailing whitespace. Can you configure your editor to automatically remove this on save?